integrations.json

```diff
@@ -766,36 +676,6 @@
       }
     ]
   },
-  {
-    "name": "Wcf",
```

Comment:
Not sure why the integrations generation is dropping these...

Reply:
Oh, that's my bad. I changed that tool's target from `net471` (or something) to `netcoreapp2.1` so it could run cross-plat, but forgot that the netfx-only integrations are hidden behind a `$if NET45` (something).

Reply:
Do you mind changing it back to `net472`? I'm fine if you sneak that into this PR.